src/commonMain/kotlin/fr/acinq/lightning/blockchain/fee/FeeEstimator.kt

```diff
@@ -15,7 +16,14 @@ interface FeeEstimator {
  * @param claimMainFeerate feerate used to claim our main output when a channel is force-closed (typically configured by the user, based on their preference).
  * @param fastFeerate feerate used to claim outputs quickly to avoid loss of funds: this one should not be set by the user (we should look at current on-chain fees).
  */
-data class OnChainFeerates(val fundingFeerate: FeeratePerKw, val mutualCloseFeerate: FeeratePerKw, val claimMainFeerate: FeeratePerKw, val fastFeerate: FeeratePerKw)
+data class OnChainFeerates(val fundingFeerate: FeeratePerKw, val mutualCloseFeerate: FeeratePerKw, val claimMainFeerate: FeeratePerKw, val fastFeerate: FeeratePerKw) {
+    constructor(feerates: Feerates) : this(
+        fundingFeerate = FeeratePerKw(feerates.slow),
```

Comment:
Do we really want to target `slow` here instead of `medium`?

Reply:
It was previously 144 blocks, but that dated back to pre-splicing. Switched to `medium` in c4d0218e.